pkg/installmanager/installmanager.go

```diff
@@ -950,6 +1003,70 @@ func (m *InstallManager) generateAssets(cd *hivev1.ClusterDeployment, workerMach
 	return nil
 }
 
+func patchLabelMachineSetManifest(manifestBytes []byte, poolsByType map[string]string, logger log.FieldLogger) (*[]byte, error) {
+	manifestBytesJson, err := yaml.YAMLToJSON(manifestBytes)
+	if err != nil {
+		return nil, errors.Wrap(err, "Error converting yaml to json")
+	}
+
+	if err != nil {
+		return nil, errors.Wrap(err, "Error converting yaml to json")
+	}
+	c, err := yamlutils.Decode(manifestBytes)
```

Comment:
We're getting this `c` _just_ for the `yamlutils.Test` on L1024? We can't do that test with gjson and get rid of this conversion entirely?